modules/installation-overview.adoc

```diff
@@ -172,6 +172,11 @@ the cluster, including the operating system itself. Each machine boots with a
 configuration that references resources hosted in the cluster that it joins.
 This configuration allows the cluster to manage itself as updates are applied.
 
+If your cluster uses a machine-provisioned infrastructure, you have the option of
```

Comment:
Please close this PR. RHEL nodes are only allowed on UPI clusters.
> In OpenShift Container Platform 4.2, you have the option of using Red Hat Enterprise Linux (RHEL) machines as compute, or worker, machines in your cluster if you use a user-provisioned infrastructure installation. 

https://docs.openshift.com/container-platform/4.2/machine_management/adding-rhel-compute.html